app/viewModels/P5/arrival/ArrivalStatusP5ViewModel.scala

```diff
@@ -79,7 +79,7 @@ object ArrivalStatusP5ViewModel {
             "movement.status.P5.action.unloadingPermission.unloadingRemarks"
           ),
           ViewMovementAction(
-            controllers.testOnly.routes.UnloadingPermissionController.getUnloadingPermissionDocument(message.messageId, arrivalId).url,
+            controllers.testOnly.routes.UnloadingPermissionController.getUnloadingPermissionDocument(arrivalId, message.messageId).url,
```

Comment:
We have a number of warnings - unused imports etc which would be good to get sorted